mmdet/core/anchor/anchor_generator.py

```diff
@@ -322,10 +324,10 @@ def single_level_valid_flags(self,
         valid_y = torch.zeros(feat_h, dtype=torch.bool, device=device)
         valid_x[:valid_w] = 1
         valid_y[:valid_h] = 1
-        valid_xx, valid_yy = self._meshgrid(valid_x, valid_y)
+        valid_xx, valid_yy = meshgrid(valid_x, valid_y)
```

Comment:
Looks like our bug: anchors and their validity flags might it be in an inconsistent order. Need a bit more investigation. You may ignore it now.